docs/mongocli/command/mongocli-auth-login.txt

```diff
@@ -75,6 +75,6 @@ Examples
 .. code-block::
 
    To start the interactive setup:
-   $ mongocli auth login
+   $ atlas auth login
```

Comment:
this should be mongocli as it a mongocli doc page

Reply:
Fixed, thanks!